core-web/libs/portlets/edit-ema/portlet/src/lib/shared/models.ts

```diff
@@ -32,11 +32,13 @@ export interface ContainerPayload {
     uuid: string;
 }
 
-interface ContentletPayload {
+export interface ContentletPayload {
     identifier: string;
-    inode: string;
-    title: string;
+    inode?: string;
+    title?: string;
+    contentType?: string;
```

Comment:
Why optionals?

Reply:
Not anymore, I made a change and now they aren't optionals